bridgetown-core/lib/bridgetown-core/concerns/site/renderable.rb

```diff
@@ -34,12 +44,21 @@ def render_docs(payload)
       end
     end
 
+    # Renders all pages
+    # @param payload [Hash] A hash of site data.
+    # @return [void]
+    # @see Bridgetown::Site::Content#site_payload
     def render_pages(payload)
       pages.each do |page|
         render_regenerated(page, payload)
       end
     end
 
+    # Regenerates a site using {Bridgetown::Renderer} and runs a new instance.
```

Comment:
I think that should be "Regenerates a document" (not site). Could probably leave off the "and runs a new instance"